pkg/kubelet/pluginmanager/pluginwatcher/README.md

```diff
@@ -22,6 +22,58 @@ should end with a DNS domain that is unique for the plugin. Each time a plugin
 starts, it has to delete old sockets if they exist and listen anew under the
 same filename.
 
+## Monitoring Plugin Connection
```

Comment:
As this is fully implemented in the dra code, would it make sense to move the documentation there? Monitoring service connection is out of scope of `pluginwatcher`, I believe.
Moving it out of `pluginmanager` code would also simplify the approval of this PR as it can be approved only by DRA approvers.

Reply:
We also need SIG Node approval for the DRA code, moving the documentation won't simplify approval. Besides, @klueska can approve also this here.

I prefer to keep documentation about kubelet plugin implementation aspects in one place. Perhaps other types will also implement this.